### ISSUE TITLE
Create missing directories

### ISSUE THREAD

**Issue by ClaireLeroyIPP** [openfisca/openfisca-survey-manager#36]
The error message 

````
AssertionError: tmp_directory does not exist: 'C:/Users/cleroy/data_collections/tmp' in ['C:\\Users\\cleroy\\.config\\openfisca-survey-manager\\config.ini']
````
should not appear when the directory is missing.
Create the missing directory instead.

**Comment by ClaireLeroyIPP** [comment]
@benjello

**Comment by maximeto** [comment]
Check double existence of ini files
Create output and tmp directories

**Comment by benjello** [comment]
Creation of output and tmp directories fixed in d4c1fe26d2f5e654b52ac885ac0687ba6c5bb1ad and 2bf0428ed2131c0a842e06235b0d12f341ed54b6
